tests/lib/kubernetes/kubernetes_base.py

```diff
@@ -83,18 +83,20 @@ def _configure_kubernetes_client(self):
         kubernetes.config.load_kube_config(self.kubeconfig)
         self.v1 = kubernetes.client.CoreV1Api()
 
-    def kubectl(self, command, check=True):
+    def kubectl(self, command, check=True, quiet=False):
```

Comment:
Should we call it `log_stdout` instead of a new name?

Reply:
Also can you separate the "fix" from the "add quiet option" in the commit so it's clear what is what?

Reply:
Yeah, probably. The separation is a little trickier, but will take a look.